samples/subsys/ipc/openamp_rsc_table/src/main_remote.c

```diff
@@ -203,7 +203,6 @@ platform_create_rpmsg_vdev(unsigned int vdev_index,
 		LOG_ERR("failed to create vdev");
 		return NULL;
 	}
-
 	/* wait master rpmsg init completion */
```

Comment:
remove this change here. I think it was added by mistake.

Reply:
Done